tldw_Server_API/app/api/v1/API_Deps/storage_quota_guard.py

```
@@ -0,0 +1,126 @@
+                "used_mb": used_mb,
+                "quota_mb": quota_mb,
+                "remaining_mb": result.get("remaining_mb", 0),
+            },
+        )
+
+    # Soft-limit warning header
+    if "soft limit" in result.get("reason", "").lower():
+        try:
+            response.headers["X-Storage-Warning"] = result["reason"]
+        except _NONCRITICAL:
+            pass
```

Ruff (0.15.6)

[warning] Use `contextlib.suppress(_NONCRITICAL)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(_NONCRITICAL): ...`

(SIM105)